var/spack/repos/builtin/packages/treelite/package.py

```diff
@@ -3,14 +3,58 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import os
 from spack import *
 
 
-class Treelite(CMakePackage):
+class Treelite(CMakePackage, PythonPackage):
```

Comment:
I don't think there's any point in extending both, is there? A lot of the settings in those build systems conflict anyway.

Reply:
I am calling some methods below that are from PythonPackage, and I'm not super familiar with Python's method scoping. Thoughts?

Reply:
`from spack import *` allows you to use `PythonPackage.install`, there's no need to subclass it as far as I know, but I could be wrong.

Reply:
Lines 57 and 60 are also calling methods from `PythonPackage`